common/app/views/fragments/nav/userAccountLinks.scala.html

```diff
@@ -4,27 +4,39 @@
 @import conf.switches.Switches.IdentityProfileNavigationSwitch
 
 @if(IdentityProfileNavigationSwitch.isSwitchedOn) {
-    <li class="menu-item js-navigation-sign-in hide-on-desktop">
+    <li class="menu-item js-navigation-sign-in">
         <a href="@Configuration.id.url/signin?INTCMP=DOTCOM_HEADER_SIGNIN"
            data-link-name="nav2 : sign in"
            class="menu-item__title">
             Sign in/up
         </a>
     </li>
 
-    <li class="menu-item u-h js-navigation-account-details hide-on-desktop">
-        <button class="menu-item__title js-navigation-toggle"
+    <li class="menu-item menu-item--user-detail u-h hide-until-desktop js-navigation-account-details">
+        <div class="menu-user">
+            @*
```

Comment:
You don't have to change it, but its more common to have something like:

`@* TODO: should investigate using the users avatar *@`